thrust/thrust/system/cuda/execution_policy.h

```diff
@@ -37,3 +37,67 @@
 #endif // no system header
 #include <thrust/system/cuda/detail/execution_policy.h>
 #include <thrust/system/cuda/detail/par.h>
+
+// define these entities here for the purpose of Doxygenating them
+// they are actually defined elsewhere
+#if 0
+THRUST_NAMESPACE_BEGIN
+namespace cuda
+{
+
+/*! \addtogroup execution_policies
```

Comment:
We are moving all Documentation strings to rst style comments, so all those should become:
```suggestion
//! \addtogroup execution_policies
```